src/SayMore/Transcription/UI/SegmentingAndRecording/OralAnnotationRecorderDlgBase.cs

```diff
@@ -1550,6 +1549,16 @@ private Rectangle GetRerecordButtonRectangleForSegmentMouseIsOver()
 				return Rectangle.Empty;
 			}
 
+			// SP-1000: Reduce minimum segment size
```

Comment:
I think you must have intended to call GetButtonRectangleForSegment for this one also?